docs/src/index.md

```diff
@@ -36,7 +36,7 @@ using Oceananigans.Units
 using Dates, CFTime
 import ClimaOcean
 
-arch = GPU()
+arch = CPU()
```

Comment:
Hmmm

Reply:
Def recommend running this on GPU or will take a long time :-D

Reply:
it was an accidental commit. I was trying on my laptop to ensure nothing errors. Fixed now